langchain/package.json

```diff
@@ -508,9 +512,11 @@
     "rollup": "^3.19.1",
     "sqlite3": "^5.1.4",
     "srt-parser-2": "^1.2.2",
+    "temp": "^0.9.4",
     "ts-jest": "^29.1.0",
     "typeorm": "^0.3.12",
     "typescript": "^5.0.0",
+    "vectordb": "^0.1.4",
     "typesense": "^1.5.3",
```

Comment:
It does seem like Yarn reorders the dependencies and creates a new lockfile, would it be possible to run `yarn` once again?